ride-share.rb

```diff
@@ -0,0 +1,95 @@
+#step 4: Total Driver's Earnings and Number of Rides
+# Use an iteration blocks to print the following answers:
+# - the number of rides each driver has given
+# - the total amount of money each driver has made
+# - the average rating for each driver
+# - Which driver made the most money?
+# - Which driver has the highest average rating?
+#optional--For each driver, on which day did they make the most money?
+
+trip_data = [
```

Comment:
This data structure worked well for you and the logic for your calculations is clear. Consider how your calculations would change had you organized the data by driver.
```ruby
trip_data = {
driver01: [{ride1}, {ride2},...],
driver02: [{ride1}, {ride2},...],
...
}

```